### PR TITLE
[PATCH v3] api: pool: clarify per thread statistics

### PR DESCRIPTION
**Pull request by psavol**

### CODE REVIEW

File: include/odp/api/spec/pool.h [hunk 1]
```diff
@@ -154,17 +154,21 @@ unsigned int odp_pool_max_index(void);
...
+ * (thread.cache_available[]) which have undefined values.
  *
- * A single call may read statistics from one to ODP_POOL_MAX_THREAD_STATS
- * threads. Set 'stats.thread.first' and 'stats.thread.last' to select the
- * threads ('first' <= 'last'). Valid values range from 0 to odp_thread_count_max() - 1.
- * A successful call fills the output array starting always from the first element
- * 'stats.thread.cache_available[0]' (='stats.thread.first').
+ * When per thread counters are enabled, application sets 'stats.thread.first' and
+ * 'stats.thread.last' to select the threads ('first' <= 'last'). A single call may read statistics
+ * from one to ODP_POOL_MAX_THREAD_STATS threads. Valid thread ID values range from 0 to
```
Comment (MatiasElo) [inline include/odp/api/spec/pool.h:165]: `#ODP_POOL_MAX_THREAD_STATS` could be used for prettier Doxygen output.

**Review by MatiasElo — APPROVED** [review]
One optional minor comment.

File: include/odp/api/spec/pool.h (hunk 1 above)
Comment (psavol) [reply include/odp/api/spec/pool.h]: Fixed in v2

**Comment by ashwinyes** [comment]
Acked-by: Ashwin Sekhar T K <asekhar@marvell.com>

**Pull request merged**